osquery/tables/system/windows/kernel_info.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <Windows.h>
+
+#include "osquery/tables/system/windows/registry.h"
```

Comment:
nit, can you move this below the global-search includes, and place a space between the two groups?